setup.py

```diff
@@ -316,7 +316,7 @@ def awscrt_ext():
     if distutils.ccompiler.get_default_compiler() != 'msvc':
         extra_compile_args += ['-Wextra', '-Werror', '-Wno-strict-aliasing', '-std=gnu99']
         if sys.platform.startswith('freebsd'):
-            extra_link_args += ['-Wl,-fatal-warnings']
+            extra_link_args += ['-Wl,-fatal-warnings', '-lexecinfo']
         else:
             extra_link_args += ['-Wl,-fatal_warnings']
```

Comment:
I think this ought to be `-falal-warnings` also or this will fail when using Clang on Linux.
I looked at both Clang and GNU linkers and neither showed using underscore in this option